eclair-core/src/test/scala/fr/acinq/eclair/integration/IntegrationSpec.scala

```diff
@@ -131,11 +109,19 @@ class IntegrationSpec extends TestKitBaseClass with BitcoindService with AnyFunS
 
   implicit val formats: Formats = DefaultFormats
 
-  override def beforeAll: Unit = {
+  override def beforeAll(): Unit = {
     startBitcoind()
+    val sender = TestProbe()
+    logger.info(s"waiting for bitcoind to initialize...")
+    awaitCond({
+      sender.send(bitcoincli, BitcoinReq("getnetworkinfo"))
+      sender.receiveOne(5 second).isInstanceOf[JValue]
+    }, max = 30 seconds, interval = 500 millis)
+    logger.info(s"generating initial blocks...")
+    generateBlocks(bitcoincli, 150, timeout = 30 seconds)
   }
 
-  override def afterAll: Unit = {
+  override def afterAll(): Unit = {
```

Comment:
Ha, we are not calling `super.afterAll()`, which would be `TestKitBaseClass.afterAll()`, which calls `TestKit.shutdownActorSystem(system)`. Maybe the explanation behind intermittent freezing after tests pass?

Reply:
Interesting, good catch!

Reply:
Done in https://github.com/ACINQ/eclair/pull/1535/commits/9a85d4b7dd44c39d854fac9e0f72991a53891719